tree/ntupleutil/v7/inc/ROOT/RNTupleImporter.hxx

```diff
@@ -209,10 +209,11 @@ private:
    RNTupleImporter() = default;
 
    std::unique_ptr<TFile> fSourceFile;
-   TTree *fSourceTree;
+   std::vector<TTree *> fSourceTrees;
 
+   /// Maps the name of the original TTree to the desired RNTuple name.
+   std::map<std::string_view, std::string> fNTupleNames;
```

Comment:
The type used as key for the `std::map` should be `std::string` instead.

Rationale: _(1)_ `std::string_view` points to a buffer owned by someone else.  Given that this buffer comes from the user (see `SetNTupleName()` or `RNTupleImporter::Create()`), we cannot make any assumptions about the lifetime of the pointed-to string; and _(2)_ the characters in the pointed-to string may change at any time, thus invalidating the map structure itself.